tests/framework/db/oci/CommandTest.php

```
@@ -33,14 +38,87 @@ public function testAutoQuoting(): void
+        );
+
+        self::assertSame(
+            0,
+            $command->execute(),
+            'Empty batch must execute as a no-op.',
+        );
+    }
+
     public function testLastInsertId(): void
     {
         $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)